### DISCUSSION TITLE
How to enable the Watch command on Windows?

### DISCUSSION THREAD

**Discussion by FlexxPlex** [Taxel/PlexTraktSync#2067] [Support]
How to enable the Watch command on Windows? Sync is working fine but I cant figure out how to enable Watch?

**Comment by glensc** [comment]
- `plextraktsync sync` - to sync
- `plextraktsync watch` - to watch

**Reply by FlexxPlex** [reply]
Where do I type those or copy paste those?

**Reply by glensc** [reply]
> Sync is working fine

same place. I have no idea what you have done.

**Comment by FlexxPlex** [comment]
I have no idea where to copy or type it. Sync is working after I installed Plextraktsync, I did not do anything just installed Plextraktsync on windows thats it. I want to change to watch now instead of sync.

**Reply by glensc** [reply]
Don't know how you installed. What instructions you followed? ANYTHING from your system?
